classes/class-kco-credentials.php

```diff
@@ -55,7 +55,7 @@ public function get_credentials_from_session() {
 
 		$credentials = array(
 			'merchant_id'   => $this->settings[ $test_string . 'merchant_id_' . $country_string ],
-			'shared_secret' => htmlspecialchars_decode( $this->settings[ $test_string . 'shared_secret_' . $country_string ] ),
+			'shared_secret' => htmlspecialchars_decode( $this->settings[ $test_string . 'shared_secret_' . $country_string ], ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401 ),
```

Comment:
Why is the `ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401` part needed? This is the default for the [PHP function](https://www.php.net/manual/en/function.htmlspecialchars-decode.php)

Reply:
![image](https://github.com/user-attachments/assets/5e14c8bc-efe2-44cf-85db-cbcf4bb679f8)